lib/App/MintTag.pm

```diff
@@ -493,6 +495,42 @@ sub _maybe_delete_source_branches ($self, $step) {
   }
 }
 
+sub _maybe_cleanup_tags ($self, $step) {
+  return unless ($step->push_tag_to && $step->cleanup_tag_days);
+
+  my $remote = $step->push_tag_to;
+  my $prefix = $step->tag_prefix;
+
+  try {
+    my $tag_spec = "refs/tags/${prefix}*";
+    my $tag_list = run_git('for-each-ref', "--format=%(refname:short)\t%(creatordate:unix)", $tag_spec);
+
+    my @tags = map {; my ($tag, $ts) = split /\t/ ; { ts => $ts, 'tag' => $tag}  }
+    split /\r?\n/, $tag_list;
+
+    my $time = time;
+
+    my $max_ts = $time - ($step->cleanup_tag_days * 86400);
+
+    for my $chunk (@tags) {
+      if ($chunk->{ts} < $max_ts) {
+        $Logger->log(["Deleting old tag %s", $chunk->{tag}]);
+        run_git('push', $remote->name, ":refs/tags/$chunk->{tag}"  );
```

Comment:
```suggestion
        run_git('push', $remote->name, ":refs/tags/$chunk->{tag}");
```

Worrying about whitespace really isn't the only thing I do.  I promise.  And if you're gonna be editing this bit anyway…